spec/services/api_login_manager_spec.rb

```diff
@@ -9,79 +9,111 @@
   # utilizarla. Es por eso que conviene usarla desde una variable con
   # let!(:password) { SecureRandom.hex }. Al crear el usuario,
   # en let!(:user) { ... }, usar esta variable.
+  EMPTY_EMAIL        = 'El email no puede estar en blanco.'.freeze
+  EMPTY_PASSWORD     = 'La contraseña no puede estar en blanco.'.freeze
+  USER_NOT_FOUND     = 'El usuario no existe'.freeze
+  WRONG_PASSWORD     = 'La contraseña es incorrecta'.freeze
+  EXTERNAL_VALIDATOR = 'El usuario ya no tiene cuota disponible'.freeze
+
   let!(:password) { SecureRandom.hex }
 
-  let!(:user) {  } # -- crear usuario con FactoryBot -- #
+  let!(:user) { FactoryBot.create(:user, password: password) } # -- crear usuario con FactoryBot -- #
 
   describe '#call' do
     context 'when user provided data is valid' do
       before do
         # Realizar un stub del ExternalValidator para que se ejecuta realmente
+        allow(ExternalValidator).to receive(:call).and_return(true)
       end
 
       subject do
         described_class.new(email: user.email, password: password).call
       end
 
       it "update user's auth_token" do
-        skip 'Una vez que definan a user, este test va a fallar. ¿Por qué?'
-        expect { subject }.to change { user.auth_token }
+        # skip 'Una vez que definan a user, este test va a fallar. ¿Por qué?'
+        # Por dos cosas: primero, un usuario creado sin parámetros recibirá una password
+        # generada que nunca sabremos cuál es, por lo que hay que crearlo con una pass
+        # conocida (la definida al comienzo del test). Segundo, si no se recarga el
+        # registro desde la base de datos, el cambio no se verá reflejado.
+        expect { subject }.to change { user.reload.auth_token }
       end
 
       it 'returns the auth_token' do
-        skip 'Implementar'
+        expect(subject).to eq(user.reload.auth_token)
       end
     end
 
     context 'when no email is provided' do
-      subject {  } # implementar
+      subject { described_class.new(password: password) }
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to eq(false)
```

Comment:
En el ejemplo de arriba el `subject` es la aplicación de `call` sobre la instancia, mientras que en éste es la instancia, y luego le mandás el mensaje `call` en el `it`. Ambas están bien, pero sería preferente usar una convención.